src/applications/vaos/new-appointment/components/DateTimeSelectPage/index.jsx

```diff
@@ -197,9 +200,12 @@ export function DateTimeSelectPage({
         </>
       )}
       <FormButtons
-        onBack={() => goBack({ routeToPreviousAppointmentPage, history })}
+        onBack={() =>
+          dispatch(routeToPreviousAppointmentPage(history, pageKey))
+        }
         onSubmit={() =>
           goForward({
+            dispatch,
             data,
             history,
             routeToNextAppointmentPage,
```

Comment:
You could remove this line, since it's not being used from the parameters in `goForward`.